### PR TITLE
EJ2MVC-440-Modified getting started document for ASP.NET MVC StockChart, Tab, TextBox and Switch controls

### PR DESCRIPTION
**Pull request by Backiaraj**

### CODE REVIEW

File: ej2-asp-core-mvc/spreadsheet/EJ2_ASP.MVC/getting-started-mvc.md [hunk 1]
```diff
@@ -113,11 +113,11 @@ Now, add the Syncfusion ASP.NET MVC Spreadsheet control in `~/Views/Home/Index.c
 {% endtabs %}
 {% endif %}
 
-Press <kbd>Ctrl</kbd>+<kbd>F5<kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app. Then, the Syncfusion ASP.NET MVC Spreadsheet control will be rendered in the default web browser.
+Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app. Then, the Syncfusion ASP.NET MVC Spreadsheet control will be rendered in the default web browser.
 
 ![ASP.NET MVC Spreadsheet Control](images/spreadsheet.PNG)
 
-> You can refer to our [ASP.NET MVC Spreadsheet](https://www.syncfusion.com/aspnet-mvc-ui-controls/spreadsheet) feature tour page for its groundbreaking feature representations. You can also explore our [ASP.NET MVC Spreadsheet example](https://ej2.syncfusion.com/aspnetmvc/Spreadsheet/DefaultFunctionalities#/material) that shows you how present and manipulate data, including editing, formulas, formatting, importing, and exporting.
```
Comment (syncsiva) [inline ej2-asp-core-mvc/spreadsheet/EJ2_ASP.MVC/getting-started-mvc.md:120]: why you are removing this notes content added by growth hacking team?

Comment (Backiaraj) [reply ej2-asp-core-mvc/spreadsheet/EJ2_ASP.MVC/getting-started-mvc.md:120]: Hi @syncsiva ,

I corrected this.

Regards,
@Backiaraj

File: ej2-asp-core-mvc/spreadsheet/EJ2_ASP.MVC/getting-started-mvc.md [hunk 2]
```diff
@@ -113,12 +113,14 @@ Now, add the Syncfusion ASP.NET MVC Spreadsheet control in `~/Views/Home/Index.c
 {% endtabs %}
 {% endif %}
 
-Press <kbd>Ctrl</kbd>+<kbd>F5<kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app. Then, the Syncfusion ASP.NET MVC Spreadsheet control will be rendered in the default web browser.
+Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app. Then, the Syncfusion ASP.NET MVC Spreadsheet control will be rendered in the default web browser.
 
 ![ASP.NET MVC Spreadsheet Control](images/spreadsheet.PNG)
 
 > You can refer to our [ASP.NET MVC Spreadsheet](https://www.syncfusion.com/aspnet-mvc-ui-controls/spreadsheet) feature tour page for its groundbreaking feature representations. You can also explore our [ASP.NET MVC Spreadsheet example](https://ej2.syncfusion.com/aspnetmvc/Spreadsheet/DefaultFunctionalities#/material) that shows you how present and manipulate data, including editing, formulas, formatting, importing, and exporting.
```
Comment (syncsiva) [inline ej2-asp-core-mvc/spreadsheet/EJ2_ASP.MVC/getting-started-mvc.md:120]: move this note below see also.

File: ej2-asp-core-mvc/stock-chart/EJ2_ASP.MVC/getting-started.md [hunk 3]
```diff
@@ -100,12 +125,11 @@ Since the JSON contains category data, set the [`valueType`](https://help.syncfu
...
 {% endhighlight %}
-{% highlight c# tabtitle="PopulateData.cs" %}
-{% include code-snippet/stock-chart/getting-started/populateData/populateData.cs %}
-{% endhighlight %}
 {% endtabs %}
 {% endif %}
 
+Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app. Then, the Syncfusion ASP.NET MVC Stock Chart control will be rendered in the default web browser.
 
+![ASP.NET MVC Stock Chart Control](images/stockchart-control.png)
```
Comment (syncsiva) [inline ej2-asp-core-mvc/stock-chart/EJ2_ASP.MVC/getting-started.md:133]: sample link missing.

**Review by syncsiva — APPROVED** [review]

**Pull request merged**